app/styles/app.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 @import 'ember-paper';
+@import "ember-paper-expansion-panel";
```

Comment:
Table styles should be there by default. See http://miguelcobain.github.io/ember-paper/#/typography